db/user.go

```diff
@@ -2,21 +2,142 @@ package db
 
 import (
 	"context"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"golang.org/x/crypto/bcrypt"
 
 	logger "github.com/sirupsen/logrus"
 )
 
+//User is a structure of the user
 type User struct {
-	Name string `db:"name" json:"full_name"`
-	Age  int    `db:"age" json:"age"`
+	ID        int    `db:"id" json:"id"`
+	FirstName string `db:"first_name" json:"first_name"`
+	LastName  string `db:"last_name" json:"last_name"`
+	Email     string `db:"email" json:"email"`
+	Mobile    string `db:"mobile" json:"mobile"`
+	Address   string `db:"address" json:"address"`
+	Password  string `db:"password" json:"password"`
+	Country   string `db:"country" json:"country"`
+	State     string `db:"state" json:"state"`
+	City      string `db:"city" json:"city"`
+	CreatedAt string `db:"created_at" json:"created_at"`
 }
 
+const (
+	getUserQuery = `SELECT * from users where id=$1`
+)
+
 func (s *pgStore) ListUsers(ctx context.Context) (users []User, err error) {
-	err = s.db.Select(&users, "SELECT * FROM users ORDER BY name ASC")
+	err = s.db.Select(&users, "SELECT * FROM users")
 	if err != nil {
 		logger.WithField("err", err.Error()).Error("Error listing users")
 		return
 	}
 
 	return
 }
+
+func (s *pgStore) GetUser(ctx context.Context, id int) (user User, err error) {
+	err = s.db.Get(&user, getUserQuery, id)
+	if err != nil {
+		logger.WithField("err", err.Error()).Error("Error selecting user from database by id " + fmt.Sprint(id))
```

Comment:
instead of `Error("Error selecting user from database by id " + fmt.Sprint(id))`
use 

`Error(fmt.Errorf("Error selecting user from database by id %s", id))`